src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -68,6 +68,15 @@ def insar_tops_burst(
     )
     results = asf_search.search(opts=opts)
 
+    ref_not_found = reference_scene not in results.__str__()
+    sec_not_found = secondary_scene not in results.__str__()
```

Comment:
I don't recall the exact interface for search results but I would guess there's a more idiomatic way to check if the scenes are in the results. If not, then I would at least suggest changing this to `str(results)`.